### PR TITLE
Add dao betatest network

### PR DESCRIPTION
**Pull request by ManfredKarrer**
This adds a mainnet genesis tx for testing as a new network type. It is only intended to be used by testers who run from source code and get in direct contact with me (as it is now real BTC I prefer to keep the test group small and focussed).

### CODE REVIEW

**Comment by ManfredKarrer** [comment]
A BSQ blockexplorer is not supported yet but we are working on it.

**Comment by ManfredKarrer** [comment]
I have started syncing btc core mainnet for the seed node. Will take a bit until that is ready. So if you want to run it before you need to provide your own seed node with --seedNodes=... can be localhost seed. 

Example local seed:
--seedNodes=localhost:3004 --baseCurrencyNetwork=BTC_DAO_BETANET --useLocalhostForP2P=true --nodePort=3004 --appName=bisq-BTC_DAO_BETANET_LOCAL_seed_3004 --daoActivated=true --fullDaoNode=true --rpcUser=xxx --rpcPassword=xxx--rpcPort=8332 --rpcBlockNotificationPort=5100

Example local Bisq app:
--seedNodes=localhost:3004 --baseCurrencyNetwork=BTC_DAO_BETANET --useLocalhostForP2P=true --nodePort=3111 --appName=bisq-BTC_DAO_BETANET_LOCAL_Ali_dao_lite --daoActivated=true

File: core/src/main/java/bisq/core/dao/state/GenesisTxInfo.java [hunk 1]
```diff
@@ -43,19 +43,25 @@
...
     private static final int TESTNET_GENESIS_BLOCK_HEIGHT = 1446300; // 2018-12-02
+    private static final Coin TESTNET_GENESIS_TOTAL_SUPPLY = Coin.parseCoin("2.5"); // 2.5M BSQ / 2.50000000 BTC
 
     private static final String DAO_TESTNET_GENESIS_TX_ID = "cb316a186b9e88d1b8e1ce8dc79cc6a2080cc7bbc6df94f2be325d8253417af1";
     private static final int DAO_TESTNET_GENESIS_BLOCK_HEIGHT = 104; // 2019-02-19
+    private static final Coin DAO_TESTNET_GENESIS_TOTAL_SUPPLY = Coin.parseCoin("2.5"); // 2.5M BSQ / 2.50000000 BTC
+
+    private static final String DAO_BETANET_GENESIS_TX_ID = "0bd66d8ff26476b55dfaf2a5db0c659a5d8635566488244df25606db63a08bd9";
+    private static final int DAO_BETANET_GENESIS_BLOCK_HEIGHT = 567405; // 2019-03-16
+    private static final Coin DAO_BETANET_GENESIS_TOTAL_SUPPLY = Coin.parseCoin("0.49998644"); // 1000 BSQ / 0.49998644 BTC
```
Comment (devinbileck) [inline core/src/main/java/bisq/core/dao/state/GenesisTxInfo.java:60]: Seems like an odd amount, but should be equal to 499,986.44 BSQ (not 1000)?

**Review by devinbileck — APPROVED** [review]
utACK

File: core/src/main/java/bisq/core/dao/state/GenesisTxInfo.java (hunk 1 above)
Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/dao/state/GenesisTxInfo.java:60]: Ah yes, I did not care much with fee and change when i created it, but it is not really important as I made amount now flexible. The real will also be not an even number if we add add up all contributions with the 2.5M from the original genesis.
Will fix the comment in another commit

**Review by sqrrm — APPROVED** [review]
utACK

I'm syncing my bitcoin node for the explorer, should be ready tomorrow from the look of it.

**Pull request merged**